### ISSUE TITLE
use descriptive alt text

### ISSUE THREAD

**Issue by discospiff** [LarryChenCode/IS7012#4]
Include concise yet descriptive alt text for each image. This should be enough to describe the image to someone who can't see it.

https://github.com/LarryChenCode/IS7012/blob/f0af076d47d8f3e9b683089838d1cdda88d931ef/Week2/RecruitCatChen4co/RecruitCatChen4co/Pages/Index.cshtml#L22
